lambdas/favourite/src/favourite-handler.ts

```diff
@@ -42,7 +48,19 @@ export class FavouriteLambda implements LambdaInterface {
     if (sessionService.hasDateExpired(sessionItem.expiryDate)) {
       throw new SessionExpiredError();
     }
+
+    const auditEventContext = this.createAuditEventContext(sessionItem);
+
+    await this.auditService.sendAuditEvent(
+      AuditEventType.REQUEST_SENT,
+      auditEventContext
+    );
     const response = await this.callExternalToy(toyBody.toy);
+    await this.auditService.sendAuditEvent(
+      AuditEventType.THIRD_PARTY_REQUEST_ENDED,
```

Comment:
It needs to be `RESPONSE_RECEIVED` as we no longer use `THIRD_PARTY_REQUEST_ENDED`

Reply:
Ah ok! I was going by what was in the TS AuditEventType but I will update